packages/vscode-extension/src/handlers.ts

```diff
@@ -1848,6 +1848,8 @@ export async function checkUpgrade(args?: any[]) {
     const input = getSystemInputs();
     if (triggerFrom?.[TelemetryProperty.TriggerFrom] === TelemetryTriggerFrom.Auto) {
       input["isNonmodalMessage"] = true;
+      core.phantomMigrationV3(input);
+      return;
     } else if (triggerFrom?.[TelemetryProperty.TriggerFrom] === TelemetryTriggerFrom.SideBar) {
       input["confirmOnly"] = true;
```

Comment:
Do we also need to apply this logic for "Teams: Upgrade project" command?

Reply:
commands will show a modal dialog not nonmodal notification.
So it will await the result

Reply:
I mean should "Teams: Upgrade project" command set "confirmOnly" to true?

Reply:
No. It will trigger the modal dialog